From 50122f3e5a56ec3ebf17e89ec870fbcee0fb5939 Mon Sep 17 00:00:00 2001
From: Jonghwa Lee <jonghwa3.lee@samsung.com>
Date: Wed, 19 Mar 2014 00:41:36 +0900
Subject: [PATCH 1301/1302] Revert "extcon: max77693: Fix bug related to
 MAX77693 irq when set ADC debounce time"

This reverts commit b64315125400f4eec9f74f7536acca3e8eeca720.
Upcoming patch will fix a bug thus all related workaournds are now purged.

Change-Id: Iab88c25ef9470a5167be73990ceb7859e076ddd8
Signed-off-by: Jonghwa Lee <jonghwa3.lee@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 drivers/extcon/extcon-max77693.c | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/drivers/extcon/extcon-max77693.c b/drivers/extcon/extcon-max77693.c
index 3cde41b4..077296d 100644
--- a/drivers/extcon/extcon-max77693.c
+++ b/drivers/extcon/extcon-max77693.c
@@ -285,14 +285,6 @@ static int max77693_muic_set_path(struct max77693_muic_info *info,
 	int ret = 0;
 	u8 ctrl1, ctrl2 = 0;
 
-	/* Set open state to path before changing hw path */
-	ret = max77693_update_reg(info->max77693->regmap_muic,
-		MAX77693_MUIC_REG_CTRL1, CONTROL1_SW_OPEN, COMP_SW_MASK);
-	if (ret < 0) {
-		dev_err(info->dev, "failed to update MUIC register\n");
-		return ret;
-	}
-
 	if (attached)
 		ctrl1 = val;
 	else
@@ -1254,7 +1246,7 @@ static int max77693_muic_probe(struct platform_device *pdev)
 	}
 
 	/* Set initial path for UART */
-	max77693_muic_set_path(info, info->path_uart, true);
+	 max77693_muic_set_path(info, info->path_uart, true);
 
 	/* Check revision number of MUIC device*/
 	ret = max77693_read_reg(info->max77693->regmap_muic,
-- 
1.8.3.2

